web/src/people.ts

```diff
@@ -375,7 +376,7 @@ export function get_user_time(user_id: number): string | undefined {
 
 export function get_user_type(user_id: number): string | undefined {
     const user_profile = get_by_user_id(user_id);
-
+    assert(user_profile !== undefined, "Unknown person in get_user_type");
     return settings_config.user_role_map.get(user_profile.role);
 }
```

Comment:
So while we expect our current user ID to always exist, I don't see a reason to expect callers of this to have checked that the user ID in question is a real one. So either that needs to be documented, or this function needs to have different error handling. 

Since this part is an API change, I think it might make sense to split it out to a prep PR.